nemo/collections/llm/recipes/llama3_8b.py

```diff
@@ -95,6 +96,17 @@ def pretrain_recipe(
     )
 
 
+def pretrain_recipe_performance(
```

Comment:
Since the factory functions can take in arguments, could we add a bool-field in the llama-8b-pretrain receip to enable comm_overlap which would be False by default?